### PR TITLE
feat: notebook title

### PR DESCRIPTION
**Pull request by ShmuelLotman**
Closes #

Allows for dynamic setting of notebook title

https://user-images.githubusercontent.com/29473622/120022845-98964180-bfb2-11eb-935c-d1a647545fdf.mov

### CODE REVIEW

File: src/flows/components/FlowPage.tsx [hunk 1]
```diff
@@ -24,12 +24,13 @@ import 'src/flows/style.scss'
...
 const FlowFromRoute = () => {
   const {id} = useParams<{id: string}>()
-  const {change} = useContext(FlowListContext)
+  const {change, flows, currentID} = useContext(FlowListContext)
 
   useEffect(() => {
     change(id)
   }, [id, change])
 
+  document.title = flows[currentID]?.name ?? 'Notebooks'
```
Comment (genehynson) [inline src/flows/components/FlowPage.tsx:33]: I think we want to mimic how dashboards handles this with https://github.com/influxdata/ui/blob/master/src/shared/utils/pageTitles.ts

Like this: https://github.com/influxdata/ui/blob/master/src/dashboards/components/dashboard_index/DashboardsIndex.tsx#L67

**Review by genehynson — CHANGES_REQUESTED** [review]
lets make this work like dashboards page!

File: src/flows/components/FlowPage.tsx (hunk 1 above)
Comment (ShmuelLotman) [reply src/flows/components/FlowPage.tsx:33]: Way to make this harder, GENE

Comment (ShmuelLotman) [reply src/flows/components/FlowPage.tsx:33]: ;)

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 46 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1550)

**Comment by ShmuelLotman** [comment]
@genehynson i could be wrong, but it seems to me the way i just pushed is close enough to the way the dashboards handles it. It uses the same underlying methods and pageTitle ends up setting the document title anyways - this was the way i thought would be least involved... lmk what you think

**Review by genehynson — APPROVED** [review]

**Pull request merged**